data/announcements/scheduled.yaml

```
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-10-01T00:00:00
+  endTime: 2021-10-16T01:00:00
+  style: "background: linear-gradient(90deg, rgb(7, 132, 111) 0%, rgb(54, 214, 183) 100%)" 
+  title: |
+    <img src="/images/announcements/kubecon-NA-2021-white.svg" style="float: right; height: 80px;" />
+    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/"> KubeCon + CloudNativeCon North America 2021</a> <em>Los Angeles, California + Virtual</em>.
+  message: |
+          5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
```

Netlify build error:
> "/opt/build/repo/data/announcements/scheduled.yaml:84:1": failed to unmarshal YAML: yaml: line 84: did not find expected key

---

This might resolve the Netlify build error.

```suggestion
    5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
```